docs/source/install-tutorial.rst

```diff
@@ -3,21 +3,36 @@
 Installation tutorial with Debian, NGinx and gunicorn
 =====================================================
 
-preliminary packages:
+Following this tutorial you'll have a running Debian+Nginx+gunicorn+bepasty (HTTP) instance using Python3 on a minimally installed Debian server.  It is strongly recommended to use HTTPS for Internet facing services - configuring Nginx for HTTPS is left to the reader.
+
+After installing a minimal Debian 10 server.
+
+Install preliminary required packages:
 
 ::
 
-  apt-get install build-essential nginx supervisor python-dev git-core python-pip python-virtualenv
+  apt-get install git-core python3-dev python3-pip virtualenv python3-virtualenv nginx supervisor 
 
 
-commands to run
+Some commands need to be run as root - others as the user bepasty.
+
+Commands to run as root:
 
 ::
 
   # add user bepasty to system
-  adduser bepasty
+  adduser bepasty --disabled-login
+  
+Commands to run as user bepasty:
+
+::
+
   # change to user bepasty
-  sudo su - bepasty
+  su - bepasty
```

Comment:
your change here only works if the user bepasty has a password set. the sudo method should also work without that.

Reply:
sudo is part of the default install & config for Ubuntu but will need to be installed and configured for a Debian minimal server install.  I have no problems either way.

Reply:
ehrm, ignore my comment. it applies for the case when one is **not** root. But at this point, the user **is** root (according to the updated instructions), so `sudo` is not needed.